packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -635,6 +645,7 @@ export const Table = <
           tableHasSubRows={hasAnySubRows}
           tableInstance={instance}
           expanderCell={expanderCell}
+          tableRowRef={tableRowRef(row)}
```

Comment:
Mybe lets not pass it if virtualization is not enabled? We dont want to collect all the refs of elements, if we not gonna use those..

Reply:
Interesting improvement! I think it is the other way - if we have virtualization, we don't need the refs. We only need them on non-virtualized tables to imperatively scroll to the row.

I tested this change on virtualized and non-virtualized tables, and it works great.